src/components/item-tag-manage/action/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { message } from 'antd';
+import { Option } from 'antd/lib/mentions';
+
+import { Button, Space } from '@src/component/atoms';
+import { WHITE0, GREY1 } from 'src/component/atoms/colors';
+import Spinner from '@src/component/organisms/spinner';
+
+import TYPES from '@src/data/item/types.json';
+import TAGS from '@src/data/item/tags.json';
+
+function Action({ onChangeAction }) {
```

Comment:
```suggestion
function ItemTagManageAction({ onChangeAction }) {
```

1. 모든 변수의 이름에는 역할이 명확히 드러나야합니다.
2. prop typing이 안 되어있습니다 ㅠ

Reply:
1820deb